doc/api.rst

```diff
@@ -46,9 +47,11 @@ Constraint-based structure learning
 
    LearnSkeleton
    LearnSemiMarkovianSkeleton
+   LearnInterventionSkeleton
```

Comment:
Can you give me a quick definition of " intervention Skeleton"?

Reply:
I added to the docstring of this class:

```
An interventional skeleton is a skeleton that is learned from observational
    and/or interventional data. The interventional skeleton is just the
    normal skeleton among the observed variables augmented with F-nodes representing
    interventions and the edges from the F-nodes. 
```